AGENTS.md

```
@@ -0,0 +1,372 @@
+
+### Directory Structure
+
+```
+├── config/           # Package configuration
+├── docs/             # Documentation (VitePress)
+├── src/              # Source code
+│   ├── Dto/          # Data Transfer Objects
+│   ├── Endpoints/    # API endpoint classes (Server, Device, User, etc.)
+│   ├── Enums/        # PHP enums for type safety
+│   ├── Facades/      # Laravel facades
+│   ├── Requests/     # Saloon request classes
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)